doc/eggex.md

```diff
@@ -311,6 +311,37 @@ Flags and translation preferences together:
 
     / digit+ ; ignorecase ; python /  # could translate to (?i)\d+
 
+In Oils, the following flags are currently supported:
+
+#### `reg_icase` / `i` (Ignore Case)
+
+Use this flag to ignore case when matching. For example, `/'foo'; i/` matches
+'FOO', but `/'foo'/` doesn't.
+
+#### `reg_newline` (Multiline)
+
+With this flag, `%end` will match before a newline and `%start` will match
+after a newline.
+
+    u'abc123\n' ~ / digit %end ; reg_newline /    # true
```

Comment:
Looks great, but I think we should insert `=` before each example line, to show it's an expression, not a command

I have tried to be consistent about examples always being valid commands.   And `= expr` is a valid command

(Also at some point we really need to make the examples runnable, but that's a bit further down the road)

Reply:
Done.